### PR TITLE
Remove per-row permissions limit

### PR DESCRIPTION
**Pull request by sam-codaio**
* The per row limit is something we are going to hit with larger instances. We should remove this
* Bumping the maximum permission batch size so that we can have more efficient permissions in cases where we do no require an extra fetch call

### CODE REVIEW

File: testing/upload_validation.ts [hunk 1]
```diff
@@ -142,7 +142,7 @@ export const Limits = {
   ColumnMatcherRegex: 300,
   NumColumnMatchersPerFormat: 10,
   NetworkDomainUrl: 253,
-  PermissionsBatchSize: 1000,
```
Comment (alexd-codaio) [inline testing/upload_validation.ts:145]: Intentional?

File: api.ts [hunk 2]
```diff
@@ -2439,26 +2430,7 @@ export function makeSyncTable<
...
-          throw new Error(
-            `Objects with ids: ${oversizedRowAccessDefinitions
-              .map(p => p.rowId)
-              .join(', ')} returned more permissions than the maximum allowed of ${MaxPermissionsPerRow} per object`,
-          );
-        }
-        return result;
-      }
-    : undefined;
+  const executeGetPermissions = wrappedExecuteGetPermissions ? wrappedExecuteGetPermissions : undefined;
```
Comment (alexd-codaio) [inline api.ts:2433]: Do you even need this? Just rename `wrappedExecuteGetPermissions` to `executeGetPermissions` and return it.

**Review by alexd-codaio — APPROVED** [review]
LGTM mod nits

File: api.ts (hunk 2 above)
Comment (sam-codaio) [reply api.ts:2433]: We do not

File: testing/upload_validation.ts (hunk 1 above)
Comment (sam-codaio) [reply testing/upload_validation.ts:145]: Yes I think we should change this as well. Double checking with some experts

**Pull request merged**